timur/jest.config.js

```diff
@@ -1,17 +1,21 @@
 module.exports = {
   globals: {
     fetch: require('node-fetch'),
-    CONFIG: {
-      project_name: 'labors'
-    }
+    TIMUR_CONFIG: {
+      project_name: 'labors',
+      magma_host: 'https://magma.test',
+    },
   },
-  testURL: 'http://www.fake.com',
+  testURL: 'http://localhost',
```

Comment:
I'm swapping out all www.fake.com with localhost because previously, our tests actually _Did_ call www.fake.com (a real website).  Pointing to localhost ensures that even if someone the netDisable nock function isn't enabled, it won't make real external network calls with random payloads.